### PR TITLE
Support for ActiveAdmin 1.0 and 1.1.0

### PR DESCRIPTION
**Pull request by rhymes**

### CODE REVIEW

**Comment by fabn** [comment]
Thanks for your PR, are you interested in becoming a maintainer of this project?

See #26

File: activeadmin-globalize.gemspec [hunk 1]
```diff
@@ -11,11 +11,11 @@ Gem::Specification.new do |s|
   s.email       = ['stefano.verna@gmail.com', 'f.napoleoni@gmail.com']
   s.homepage    = 'http://github.com/fabn/activeadmin-globalize'
   s.summary     = 'Handles globalize translations'
-  s.description = 'Handles globalize translations in ActiveAdmin 0.6.3 and Rails 3.2.x'
+  s.description = 'Handles globalize translations in ActiveAdmin 1.0 and Rails 4.x-5.x'
 
   s.files = Dir['{app,config,db,lib}/**/*'] + %w(MIT-LICENSE README.md)
 
-  s.add_dependency 'activeadmin'
+  s.add_dependency 'activeadmin', '>= 1.0', '< 1.1'
```
Comment (besi) [inline activeadmin-globalize.gemspec]: @rhymes Here you could do `< 1.2` to support ActiveAdmin 1.1

**Comment by besi** [comment]
I did try your branch with Activeadmin 1.1 and it worked fine so far:

screen shot 2017-10-02 at 22 11 57

screen shot 2017-10-02 at 22 12 35

screen shot 2017-10-02 at 22 12 51

screen shot 2017-10-02 at 22 12 43

The only problem I currently faced is that I sometimes have only a German flag and sometimes an English and a German flag for entries that, in both cases, only had an English translation.

**Comment by rhymes** [comment]
@besi you're right. I've allowed AA 1.1. The issue is that this branch (as the upstream branch) does not remotely pass any test on Travis :-(

**Comment by fabn** [comment]
Thanks for your PR, since you and others reported it to work I've merged it and I'm going to release it as version `1.0.0.pre`

I'm about to push it to rubygems.

**Comment by fabn** [comment]
Version 1.0.0.pre is [out](https://rubygems.org/gems/activeadmin-globalize/versions/1.0.0.pre)

**Pull request merged**